packages/@uppy/provider-views/src/Browser.tsx

```diff
@@ -22,52 +45,84 @@ function ListItem (props) {
     i18n,
     validateRestrictions,
     getNextFolder,
-    columns,
     f,
   } = props
 
   if (f.isFolder) {
-    return Item({
-      columns,
+    return Item<M, B>({
       showTitles,
       viewType,
       i18n,
       id: f.id,
       title: f.name,
       getItemIcon: () => f.icon,
       isChecked: isChecked(f),
-      toggleCheckbox: (event) => toggleCheckbox(event, f),
+      toggleCheckbox: (event: Event) => toggleCheckbox(event, f),
       recordShiftKeyPress,
       type: 'folder',
-      isDisabled: isChecked(f)?.loading,
+      // TODO: when was this supposed to be true?
+      isDisabled: false,
```

Comment:
That looks like a potential breaking change, maybe best not to do it and only keep the TODO

Reply:
It's not a breaking change because the previous was done incorrectly, making it always false.